models/unittest/unit_tests.go

```diff
@@ -96,7 +96,15 @@ func AssertExistsAndLoadMap(t assert.TestingT, table string, conditions ...any)
 // GetCount get the count of a bean
 func GetCount(t assert.TestingT, bean any, conditions ...any) int {
 	e := db.GetEngine(db.DefaultContext)
-	count, err := whereOrderConditions(e, conditions).Count(bean)
+	for _, condition := range conditions {
```

Comment:
If the `limit` implementation of xorm is not fixed, this code will run into a bug too. But in our cases, we just don't use `limit` when writing test cases so the bug is not triggered.